### ISSUE TITLE
Tutorials for motoman_driver package not correctly indexed

### ISSUE THREAD

**Issue by shaun-edwards** [rosindex/rosindex.github.io#90]
The tutorials for the `motoman_driver` package aren't correctly indexed (see [here](http://rosindex.github.io/p/motoman_driver/github-ros-industrial-motoman/#hydro)).  The tutorials do exist (see [here](http://wiki.ros.org/motoman_driver/Tutorials), but they don't seem to be counted.

This might be a broader issue, but I thought I would give a specific example.

**Comment by jbohren** [comment]
Yeah, none of the tutorials on the wiki are currently indexed. My opinion is that any tutorials should be put in buildable packages, and not on the ROS wiki. Then the tutorials should be described in a statically-analyzable way like in a package's manifest or a file linked from the package's manifest.

You're right that this is currently misleading though, so  there should at least be a link to the wiki tutorials page.

**Comment by jbohren** [comment]
I should be able to work out most wiki tutorials by downloading this blob:

http://wiki.ros.org/?action=titleindex

**Comment by jbohren** [comment]
The titleindex parsing works pretty well: http://rosindex.github.io/p/motoman_driver/github-ros-industrial-motoman/#hydro-tutorials

It'd be nice if there was a standard layout for different versions of tutorials for different distros, but that won't really matter if people start putting them entirely in version control.

Some do `/Tutorials/SomethingCool/hydro` and others do `/Tutorials/hydro/SomethingCool`. We could probably pick this up, too, but for now, at least everything relevant is displayed there.
